lib/srv/desktop/tdp/proto.go

```diff
@@ -1631,6 +1636,44 @@ func decodeSharedDirectoryTruncateResponse(in io.Reader) (SharedDirectoryTruncat
 	return res, err
 }
 
+// LatencyStats is used to report the latency of the connection(s) to the web UI.
+type LatencyStats struct {
+	BrowserLatency uint32
+	DesktopLatency uint32
```

Comment:
What do you think of calling these fields `Client` and `Server`? 
Both `latency.Statistics` and the frontend's `LatencyDiagnostic` already use these terms, so keeping them consistent might be more convenient (we wouldn't have to convert the object on the JS side).

Reply:
Makes sense, renamed